.circleci/config.yml

```diff
@@ -0,0 +1,99 @@
+version: 2
+
+defaults: &defaults
+  working_directory: ~/project/yarn
+  docker:
+    - image: yarnpkg/dev:latest
+
+jobs:
+  install:
+    <<: *defaults
+    steps:
+      - checkout
+      - restore_cache:
+          key: yarn-{{ .Branch }}-{{ checksum "yarn.lock" }}
+      - run:
+          name: Install Dependencies
+          command: yarn install
+      - save_cache:
+          key: yarn-{{ .Branch }}-{{ checksum "yarn.lock" }}
+          paths:
+            - .cache/yarn
+      - persist_to_workspace:
+          root: ~/project
+          paths:
+            - yarn
+  test:
+    <<: *defaults
+    steps:
+      - checkout
+      - attach_workspace:
+          at: ~/project
+      - run:
+          name: Tests
+          command: |
+            node -v
+            if [ "$CIRCLE_BRANCH" == 'master' ]; then
+              ./scripts/set-dev-version.js
+            fi;
+            yarn lint
+            yarn test-ci -- -- --maxWorkers 3
+            yarn check-lockfile
+  build:
+    <<: *defaults
+    steps:
+      - checkout
+      - attach_workspace:
+          at: ~/project
+      - run:
+          name: Build distribution
+          command: |
+            node -v
+            if [ "$CIRCLE_BRANCH" == 'master' ]; then
+              ./scripts/set-dev-version.js
+            fi;
+            yarn build-dist
+            node ./scripts/build-webpack.js
```

Comment:
There's no call to `build-webpack.js` in the old one... Why do we need it here? How was the old one doing it?

Reply:
Hmm, I will check. Actually we may not need to run this command here.

Reply:
It is included in the processing of `yarn build-dist`